Python/Scrap_Latest_Torrents/README.md

```diff
@@ -0,0 +1,20 @@
+## Scrap Latest Torrents
+
+This script downloads the torrents in our Torrent client from the website https://rarbg.to/ from an entered label.
+
+### Requirements
+
+1.  You must have a torrent client, for example: uTorrent, BitTorrent, etc
+
+### Setup
+
+1.  Create a Virtual Environment.
+2.  Install the requirements by using  `pip3 install -r requirements.txt`
+
+### Running a File
+
+1.  Run the Script  `python scrap_latest_torrents.py`
+2.  Enter the label to search, example: Mr Robot.
+3.  The number of Torrents found will be displayed, you must confirm the download
+
+![How to use the script](https://i.ibb.co/PD3sPLf/Scrap-Latest-Torrents.png)
```

Comment:
Hey, can you add an Author Subheading as well, we would love to have your name here.

Reply:
Add Author